CHANGELOG.md

```diff
@@ -90,6 +90,7 @@
 * If you set a subscription on a link in flexible sync, the server would not know how to handle it ([#5409](https://github.com/realm/realm-core/issues/5409), since v11.6.1)
 * If a case insensitive query searched for a string including an 4-byte UTF8 character, the program would crash. (Core upgrade)
 * Added validation to prevent adding a removed object using Realm.Add. (Issue [#3020](https://github.com/realm/realm-dotnet/issues/3020))
+* Added test to validate that deleting a realm containing frozen objects does not throw if the frozen object's realm and the live realm have been disposed. (Issue [#2808](https://github.com/realm/realm-dotnet/issues/2808))
```

Comment:
We don't add info about new tests to the changelog. We use it for user-facing changes, so I would remove this